src/main/res/values/strings.xml

```diff
@@ -118,10 +118,10 @@
     <!-- Menu Prefs -->
 
     <string name="pref_menus_post_context_items_key" translatable="false">pref_menus_post_context_items_2</string>
-    <string name="pref_menus_post_context_items_title">Post Context Menu Items</string>
+    <string name="pref_menus_post_context_items_title">Menu Items for Post Context</string>
```

Comment:
This doesn't make any sense, it's the items in the post context menu. Original wording was fine.

Reply:
The original suggests they are post context.